src/cls/rgw/cls_rgw_types.h

```diff
@@ -340,6 +340,10 @@ struct cls_rgw_obj_key {
   cls_rgw_obj_key(const std::string &_name) : name(_name) {}
   cls_rgw_obj_key(const std::string& n, const std::string& i) : name(n), instance(i) {}
 
+  operator std::string() const {
```

Comment:
would fmt be infinitesimally faster?

Reply:
Probably! I need to start using it habitually.